<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1493</width>
    <height>681</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>899</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NTMicroDrive</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>9900</width>
     <height>640</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="4">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBoxSpeed">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>Speed [PPS]</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinAndMaxSize</enum>
                 </property>
                 <item>
                  <widget class="QSpinBox" name="speedspinBox">
                   <property name="minimumSize">
                    <size>
                     <width>90</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>13</pointsize>
                     <weight>50</weight>
                     <italic>false</italic>
                     <bold>false</bold>
                     <underline>false</underline>
                     <strikeout>false</strikeout>
                    </font>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="maximum">
                    <number>250</number>
                   </property>
                   <property name="singleStep">
                    <number>8</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="speedSlider">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>250</number>
                   </property>
                   <property name="singleStep">
                    <number>8</number>
                   </property>
                   <property name="pageStep">
                    <number>8</number>
                   </property>
                   <property name="sliderPosition">
                    <number>0</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                   </property>
                   <property name="tickInterval">
                    <number>384</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="speedChangeButton">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QGroupBox" name="groupBoxPosition">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Position [microsteps]</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="1" column="0">
                  <widget class="QSpinBox" name="positionspinBox">
                   <property name="minimumSize">
                    <size>
                     <width>90</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>13</pointsize>
                     <weight>50</weight>
                     <italic>false</italic>
                     <bold>false</bold>
                     <underline>false</underline>
                    </font>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="minimum">
                    <number>-16000</number>
                   </property>
                   <property name="maximum">
                    <number>16000</number>
                   </property>
                   <property name="singleStep">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSlider" name="positionSlider">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="minimum">
                    <number>-16000</number>
                   </property>
                   <property name="maximum">
                    <number>16000</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                   </property>
                   <property name="tickInterval">
                    <number>2000</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="posChangeButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="groupBoxDirection">
          <property name="title">
           <string>Direction</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButtonDirectionAntiClockwise">
             <property name="text">
              <string>anti-clockwise</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonDirectionClockWise">
             <property name="text">
              <string>clockwise</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonDirectionStop">
             <property name="text">
              <string>stop</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxAcceleration">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Control Options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonPositionCtrl">
             <property name="text">
              <string>Position Control</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonSpeedCtrl">
             <property name="text">
              <string>Speed Control</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxControlSelection">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Motor Control</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButtonResetInputs">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>Reset Inputs</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonEnableMotorCtrl">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>Enable Motor Control</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>BVDD</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="PlotWidget" name="graphicsViewBVDD"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Temperature</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="PlotWidget" name="graphicsViewTemperature"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Rotor Speed</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="PlotWidget" name="graphicsViewRotorSpeed"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="3">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="labelErrorLED">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/led/resources/red-led-off.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>  Error  </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="4">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="labelOverCurrentLED">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/led/resources/red-led-off.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>  Over Current  </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="5">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="labelOverTemperatureLED">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/led/resources/red-led-off.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>  Over Temperature  </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLCDNumber" name="currentPositionLCD">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="digitCount">
             <number>6</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Current Position</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="6">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="labelTargetOnlineLED">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/led/resources/green-led-off.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>  Target Online  </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="ImageButton">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>622</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>622</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/Image/resources/NT_BannerSW31.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuClose">
    <property name="title">
     <string>App</string>
    </property>
   </widget>
   <addaction name="menuClose"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
